core/src/main/scala/org/apache/spark/memory/StaticMemoryManager.scala

```diff
@@ -104,6 +104,8 @@ private[spark] class StaticMemoryManager(
 
 private[spark] object StaticMemoryManager {
 
+  private val MIN_MEMORY_BYTES = 450 * 1024 * 1024
```

Comment:
Should this not be connected to some common constant somewhere? Is 450MB a fixed value checked elsewhere? Failing more clearly is fine of course.